index.html

```diff
@@ -0,0 +1,237 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="css/index.css">
+    <script src="/scripts/index.js" defer></script>
+</head>
+
+<body>
+    <div class="aside-wrapper" id="aside-wrapper">
+        <div class="aside">
+            <button class="aside__button" id="aside-button">
+                <img src="../images/aside-button.svg" alt="close" class="aside__button-image">
+            </button>
+            <ul class="aside__list">
+                <li class="aside__list-element">
+                    <button class="aside__list-element-button aside__list-element-button-skills" id="aside-list-element-button-skills">Skills</button>
+                </li>
+                <li class="aside__list-element">
+                    <button class="aside__list-element-button aside__list-element-button-portfolio" id="aside-list-element-button-portfolio">Portfolio</button>
+                </li>
+                <li class="aside__list-element">
+                    <button class="aside__list-element-button aside__list-element-button-video" id="aside-list-element-button-video">Video</button>
+                </li>
+                <li class="aside__list-element">
+                    <button class="aside__list-element-button aside__list-element-button-price" id="aside-list-element-button-price">Price</button>
+                </li>
+                <li class="aside__list-element">
+                    <button class="aside__list-element-button aside__list-element-button-contacts" id="aside-list-element-button-contacts">Contacts</button>
+                </li>   
+            </ul>
+        </div>
+    </div>
+    <div class="home">
+        <header class="header">
+            <nav class="header__items">
+                <img src="images/logo.svg" alt="logo" class="header__items-logo">
+                <div class="header__items-right-part">
+                    <ul class="header__items-right-part-list" id="header-items-right-part-list">
+                        <li class="header__items-right-part-list-element">
+                            <a href="#" class="header__items-right-part-list-element-link header__items-right-part-list-element-link-skills">Skills</a>
```

Comment:
You don't need to add separate classes for JS selectors, using id is better in that situations. By the way, class name is too long, you could write something like "header__link". We already know which link is it, if it is inside the header, right?